src/views/Calendar.vue

```diff
@@ -99,7 +99,7 @@ export default {
     },
     //現在の日時を判定する処理
     isToday: function() {
-      let date = moment();
+      const date = moment();
       if (this.getCalendar === date) {
         return true;
       }
```

Comment:
これはif文あるあるなのですが、 ` this.getCalendar === date` ってbooleanの評価値が出ているので
それごとreturnするだけで良いです

```suggestion
      return this.getCalendar === date
```